static/app/views/admin/adminSettings.tsx

```diff
@@ -6,6 +8,36 @@ import AsyncView from 'sentry/views/asyncView';
 
 import {getOption, getOptionField} from './options';
 
+const manualDetectionOptions = [
+  'performance.issues.n_plus_one_db.problem-creation',
+  'performance.issues.n_plus_one_db_ext.problem-creation',
+  'performance.issues.n_plus_one_db.count_threshold',
+  'performance.issues.n_plus_one_db.duration_threshold',
+  'performance.issues.file_io_main_thread.problem-creation',
+];
+
+// For rollout options backed by `AutoRegisterOptionBackedRolloutFeatureHandler`
+export const autoRegisterDetectorOptions = [
+  {
+    optionRoot: 'performance.issues.consecutive_db',
+    detectorName: 'Consecutive DB',
+  },
+  {
+    optionRoot: 'performance.issues.n_plus_one_api_calls',
+    detectorName: 'N+1 API Calls',
+  },
+  {
+    optionRoot: 'performance.issues.compressed_assets',
+    detectorName: 'Compressed Assets',
+  },
+  {
+    optionRoot: 'performance.issues.slow_db_query',
+    detectorName: 'Slow DB Span',
+  },
+];
```

Comment:
Nit: Confusing that it's called `optionRoot` here, and `namespace` in the backend PR. I don't know which one is more correct, but it'd be nice to keep them the same!

Reply:
namespace, let's keep it.